awscli/formatter.py

```diff
@@ -91,10 +92,11 @@ def _format_response(self, command_name, response, stream):
         # the response will be an empty string.  We don't want to print
         # that out to the user but other "falsey" values like an empty
         # dictionary should be printed.
-        if response:
```

Comment:
Makes sense. By the way, will we also document this philosophy somewhere? Such as in the code as a comment paragraph?